src/charts/legend.coffee

```diff
@@ -89,11 +95,14 @@ Ember.Charts.Legend = Ember.Mixin.create
       col = i % numItemsPerRow
       row = Math.floor(i / numItemsPerRow)
       items = if isSingleRow then numAllItems else numItemsPerRow
-      dx = col * legendItemWidth - items/2 * legendItemWidth + 1/2
+      #Adding averageLegendLabelWidth/2 is a magic number that is about right
```

Comment:
Who sets `averageLegendLabelWidth`? It looks like it's not computed inside the charts library, which might mess up existing users.